nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
Yes, but see my comment about half the world stopping if it didn't work.  This is one of those areas where it's okay to ignore the spec and get on with work.